NodeJS/NodeMailer/README.md

```diff
@@ -0,0 +1,37 @@
+# Node Contact Form
+
+Simple Node.js/Express app using Nodemailer to send emails
+
+- Please add your own SMTP info for it to work
+
+### Version
+
+1.0.0
+
+## Install Dependencies
+
+```bash
+1. express
+2. body-parser
+3. express-handlebars
+4. nodemailer
+5. path
+```
+
+
+
+## Quick Start
+
+```bash
+1. git clone https://github.com/abhijeet007rocks8/Dev-Scripts.git
+2. cd NodeJs
+3. cd ModeMailer
+4. npm i
+5. nodemon app
+6. Goto browser and do localhost:5000
+```
```

Comment:
@mayank-del, Just a few more changes here, 
only add the commands in the code-block, and remove all these points from the code-block.

Reply:
@manisha-27  maam ,now merge it ,I did it, now only 1 day left.